rust/kcl-lib/src/parsing/parser.rs

```diff
@@ -3254,6 +3255,14 @@ mod tests {
         assert_eq!(err.message, "Unexpected end of file. The compiler expected )");
     }
 
+    #[test]
+    fn kw_call_as_operand() {
+        let tokens = crate::parsing::token::lex("f(x = 1)", ModuleId::default()).unwrap();
```

Comment:
Should this be `val = f(x = 1) + 1`? And if so, I don't think we need the snapshot test since we only want to see that it parses, right?